src/fsharp/SyntaxTreeOps.fs

```diff
@@ -397,6 +397,14 @@ let (|Attributes|) synAttributes =
 let rangeOfNonNilAttrs (attrs: SynAttributes) =
     (attrs.Head.Range, attrs.Tail) ||> unionRangeWithListBy (fun a -> a.Range)
 
+let rec skipParenTypes synType =
+    match synType with
+    | SynType.Paren (innerType, _) -> skipParenTypes innerType
+    | _ -> synType
+
+let rec (|SkipParenTypes|) synType =
```

Comment:
```suggestion
let (|SkipParenTypes|) synType =
```
`rec` shouldn't be necessary right?

Reply:
I was needed before the logic was extracted to the function above, thanks!